test/http_router_test.exs

```diff
@@ -0,0 +1,113 @@
+defmodule HTTPServerTest.Router do
+  use ExUnit.Case
+  alias HTTPServer.Router
+  alias HTTPServer.Request
+  alias HTTPServer.Response
+  # doctest HTTPServer
+
+  defmodule HttpRouterTest.MockRoutes do
+    alias HTTPServer.Request
+    alias HTTPServerFixture.SimpleGet
+    alias HTTPServerFixture.TestGetWithBody
+    alias HTTPServerFixture.SimplePost
```

Comment:
These are still `HTTPServerFixture` code.  You could just make mock routes right in this file to start.

```elixir
defmodule HTTPServer.TestGetHandler do
alias HTTPServer.Request
  @behaviour HTTPServer.Handler

  @impl HTTPServer.Handler
  def handle(%Request{method: "GET"} = _req) do
    {200, "this is a mocked get request"}
  end
end
```